scripts/bootstrap.sh

```diff
@@ -67,10 +67,16 @@ download() {
     local BINARY_FILE=$1
     local URL=$2
     echo "===> Downloading: " "${URL}"
-    wget "${URL}" || rc=$?
-    tar xvzf "${BINARY_FILE}" || rc=$?
-    rm "${BINARY_FILE}"
-    if [ -n "$rc" ]; then
+    if [[ $(command -v wget) ]]; then
+        wget "${URL}" || rc=$?
+        tar xvzf "${BINARY_FILE}" || rc=$?
+        rm "${BINARY_FILE}"
+    else
+        echo "wget not available, falling back to curl"
+        curl -s -L "${URL}" | tar xz || rc=$?
+    fi
+
+    if [[ -n "$rc" ]]; then
```

Comment:
Since this is getting complicated again... we would benefit from a script comment explaining why wget is preferred over curl.